pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/api.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package scaffolds
 
 import (
```

Comment:
Hi @sarthaksarthak9 

See that it is failing in the lint
We need to run `make lint-fix` then the issues will be solved.